src/components/shared/card/Card.tsx

```diff
@@ -1,20 +1,14 @@
 import classNames from "classnames";
 import * as React from "react";
-
 import "./Card.scss";
 
-interface ICardProps extends React.Props<any> {
+interface CardProps {
   className?: string;
+  children: React.ReactNode;
 }
```

Comment:
Hellooo,

In a previous PR I have changed the interface removing the prefix I too as you have done but as I see it is a naming convention used by repo owners so I think that is better to leave as is. What you think?

```suggestion
interface ICardProps {
  className?: string;
  children: React.ReactNode;
}
```

Reply:
As the repo owner, I can say that it's better without the `I` prefix, but don't focus too much on that right now, as it's something that can be easily replaced later. 

Thank you!

Reply:
Hahahha Sorry I didn't see that you were the repo owner

Reply:
No problem xD :smile: